Sources/MapboxDirectionsCLI/main.swift

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/swift
+
+import Foundation
+import MapboxDirections
+import SwiftCLI
+
+
+let match = ProcessCommand< MapMatchingResponse, MatchOptions > (name: "match",
+                                                                 shortDescription: "Command to process Map Matching Data")
+let route = ProcessCommand< MapboxDirections.RouteResponse, RouteOptions>(name: "route",
+                                                                          shortDescription: "Command to process Routing Data")
+
+CLI(name: "MapboxDirectionsCLI",
```

Comment:
It’s a bit redundant to name a command line tool with “CLI”. Would it be possible to shorten it to `MapboxDirections` or give it a more conventional command-line name like `mapbox-directions-swift`?